### PR TITLE
add simple wpt demo

### PR DESCRIPTION
**Pull request by fippo**
I couldn't think of anything useful so I'm just outputting the "frame" (including padding) via `console.table`

### CODE REVIEW

**Comment by jlaine** [comment]
Feel free to completely gut the HTML + JS I wrote, it's not my finest work. In other words : you don't necessarily need a separate wpt.html, I'm fine with replacing index.html

File: htdocs/lib.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+            resolve(JSON.parse(message.data))
+        });
+    }));
+    await pc.setRemoteDescription(answer);
+    return ws;
+}
+
+// Determine if a packet is an RTCP packet.
+// See https://tools.ietf.org/html/rfc5761#section-4
+function isRTCP(packet) {
```
Comment (jlaine) [inline htdocs/lib.js:39]: This makes me think you maybe want the WebRTC endpoint to echo back RTCP too?

Comment (fippo) [reply htdocs/lib.js:39]: I tried but somehow that ended up blocking. Then I found a bug in the JS client code here (missing `& 0x7f` on the payloadType here and things started working. Will update :-)

**Comment by fippo** [comment]
updated. I must resist adding something that sends a copy of the stun packets to the client :-)

**Comment by jlaine** [comment]
Let me know when you're happy with me hitting the merge button, or even move this repo to shared ownership

**Comment by fippo** [comment]
:raised_hands: -- long-term plan would be to make this repository obsolete and the tests available in web-platform-tests but lets use it as a staging ground for now

**Pull request merged**